site.yml

```diff
@@ -326,6 +326,7 @@
         job_template: cf-deploy
         target_environment_name: trial
         platform: aws
+        extra_options: "WEB_ACCESS_CIDRS=0.0.0.0/0"
```

Comment:
Do we really want to open stage-trial and trial to the whole world? I though we had a list of IPs from our trialists...